shell/platform/fuchsia/flutter/platform_view.cc

```diff
@@ -35,6 +35,76 @@ static constexpr char kAccessibilityChannel[] = "flutter/accessibility";
 static constexpr char kFlutterPlatformViewsChannel[] = "flutter/platform_views";
 static constexpr char kFuchsiaShaderWarmupChannel[] = "fuchsia/shader_warmup";
 
+static constexpr char kInputActionKey[] = "inputAction";
```

Comment:
It feels like we should be able to write unit tests for the logic in this PR. Is there something that prevents us from adding tests to platform_view_unittest.cc?

Reply:
Yes, except platform_view.cc isn't very test-friendly as is. I had planned to do a follow-up where I factor the text editing delegate out into a separate compilation unit and then add a unit test for that. If you think this is introducing too many moving parts, happy to do the refactor together with this change, or first.

Reply:
This is now done. PTAL.